VideoLocker/src/main/java/org/edx/mobile/util/AppStoreUtils.java

```diff
@@ -0,0 +1,60 @@
+package org.edx.mobile.util;
+
+import android.content.ActivityNotFoundException;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.view.View;
+import android.widget.Toast;
+
+import org.edx.mobile.BuildConfig;
+import org.edx.mobile.R;
+
+/**
+ * Utility class for interacting with an app store, or the
+ * Play Store specifically.
+ */
+public final class AppStoreUtils {
+    // Make this class non-instantiable
+    private AppStoreUtils() {
+        throw new UnsupportedOperationException();
+    }
+
+    /**
+     * Open an app store to display the app.
+     *
+     * @param context A Context for starting the new Activity.
+     */
+    public static void openAppInAppStore(final Context context) {
+        Intent intent = new Intent(Intent.ACTION_VIEW);
+        intent.addCategory(Intent.CATEGORY_BROWSABLE);
+        intent.setData(Uri.parse("market://details?id=" + BuildConfig.APPLICATION_ID));
+        try {
+            context.startActivity(intent);
+        } catch (ActivityNotFoundException e) {
+            intent.setData(Uri.parse("https://play.google.com/store/apps/details?id=" +
+                            BuildConfig.APPLICATION_ID));
```

Comment:
Yes, we should definitely have a config flag for the app store URL.

Additionally, for the official edX app, we should explicitly open the Play Store app. Users can install other apps that intercept "market://" URLs, but in our case we don't want them to open a different app store because we know they need to use Google Play. See: http://stackoverflow.com/a/28090925 for a potential solution.